experimental/ner_spancat_compare/project.yml

```
@@ -24,6 +24,18 @@ description: |
   | cell_line | 3.29        | 1.91                 | 1.04                     |
   | RNA       | 2.73        | 2.68                 | 1.28                     |
 
+  The table above shows the average span length for each span type, and their
+  corresponding distinctiveness characteristics. The latter is computed using
+  the KL-divergence of the span's token distribution with respect to the overall
+  corpus's. The higher the number is, the more distinct the tokens are compared to the
+  rest of the corpus.
+
+  These characteristics can give us a good intuition as to how well the SpanCat
+  model identify the correct spans. In the case of GENIA, the entities
```

```suggestion
  model identifies the correct spans. In the case of GENIA, the entities
```
 \+ readme update ;-)